jvm-libs/generic/persistence/db/src/testFixtures/kotlin/net/consensys/zkevm/persistence/db/test/CleanDbTestSuiteParallel.kt

```diff
@@ -26,6 +26,8 @@ abstract class CleanDbTestSuiteParallel {
 
   private fun createDataSource(databaseName: String): DataSource {
     return PGSimpleDataSource().also {
+      it.ssl = false
+      it.gssEncMode = "disable"
```

Comment:
@VGau I added the above changes as an attempt to fix the randomly failing issue of tx-exclusion-api integration test, these are the closest potential solution I could find (from logs and online materials), I tested them in my another PR and I couldn't be able to reproduce the failure after this change, so could we add them in this PR to see hopefully if it can resolve the random failure issue of tx-exclusion-api integration test